doc/sphinx/addendum/miscellaneous-extensions.rst

```diff
@@ -7,20 +7,24 @@ of program refinements. To use the Derive extension it must first be
 required with ``Require Coq.derive.Derive``. When the extension is loaded,
 it provides the following command:
 
-.. cmd:: Derive @ident__1 SuchThat @one_term As @ident__2
-
-   :n:`@ident__1` can appear in :n:`@one_term`. This command opens a new proof
-   presenting the user with a goal for :n:`@one_term` in which the name :n:`@ident__1` is
-   bound to an existential variable :g:`?x` (formally, there are other goals
+.. cmd:: Derive @open_binders in @type as @ident
+         Derive @open_binders SuchThat @type As @ident
+
+   where :n:`@open_binders` declares a list of declarations of the form
+   :g:`@ident__i : @type__i` which can appear in :n:`@type`. This
```

Comment:
Will you generate error messages if the open_binders have something other than `@ident : @type`?

Reply:
Yes, there is an error message if they don't have the expected form.